lib/graphql/schema_comparator/changes.rb

```diff
@@ -1,6 +1,21 @@
 module GraphQL
   module SchemaComparator
     module Changes
+      module SafeTypeChange
+        def safe_change?(old_type, new_type)
+          if !old_type.kind.wraps? && !new_type.kind.wraps?
+            old_type.name == new_type.name
+          elsif old_type.kind.list?
+            new_type.kind.list? && safe_change?(old_type.of_type, new_type.of_type)
```

Comment:
can this be simplified with this maybe?

```ruby
elsif old_type.kind.list? && new_type.kind.list?
  safe_change?(...)
```

Reads a bit better to me. would it still be valid?

Reply:
Re-structured both conditionals a bit. I think it reads better along with your suggestion.

Reply:
Thank you guys